040-SDK/103-update.mdx

```diff
@@ -515,7 +458,7 @@ The way to treat this error depends on your application. You might be able to re
 
 ## Updating multiple records
 
-Multiple record update operations can be performed in a single call using the [Transaction](/docs/sdk/transaction) API.
+If you have multiple records to update, you can send them in a single call using the [`/transactions` endpoint](/docs/sdk/transaction). The Xata transactions API allows bulk updates where specified fields of records are modified. These updates require an ID and will fail if the IDs aren't found in the database. For more details, refer to the [transactions updates documentation](https://xata.io/docs/sdk/transaction#updates).
```

Comment:
This is _partially_ true: there is the `upsert` flag that specifies the behavior.
https://xata.io/docs/sdk/transaction#updates

The default is false, which means indeed the updates will fail if the id doesn't already exist.
With upsert set to true, a new record will be created if no record is found with the provided ID.